web-ng/src/app/services/router/router.service.ts

```diff
@@ -33,6 +33,10 @@ export class NavigationService {
   private static readonly OBSERVATION_ID_FRAGMENT_PARAM = 'o';
   static readonly LAYER_ID_NEW = 'new';
   static readonly OBSERVATION_ID_NEW = 'new';
+  static readonly PROJECT_ID_NEW = 'new';
+  static readonly PROJECT_ID = 'projectId';
+  static readonly DEFAULT_PROJECT_ROUTE = 'project';
+  static readonly SIGN_IN_ROUTE = 'signin';
```

Comment:
Same here - this isn't a complete route, just a segment. Please rename accordingly.